### DISCUSSION TITLE
Breadcrumbs are unpredicable when using site example

### DISCUSSION THREAD

**Discussion by developdeez** [gatsbyjs/gatsby#28697] [Help]
My app uses gatsby with a Layout component I put on all of my pages.
I have the useBreadcrumbs in the Layout taking the location and a sent in title.
I noticed my page rerenders multiple times sending both the last and current titles to useBreadcrumb.
I also noticed this doesn't happen without this plugin.

There appears to be no working examples online.

How do you make this actually work on pages?

Expected: Breadcrumbs in order
Actual: No predictable Breadcrumbs

**Comment by vladar** [comment]
Can you clarify what plugin are you referring to? There is no plugin in this monorepo with the `useBreadcrumb` hook. You should probably raise this question in that plugin repository instead.

**Reply by developdeez** [reply]
Created an updated ticket:https://github.com/gatsbyjs/gatsby/issues/28698

**Reply by developdeez** [reply]
Im trying to use this one: https://www.gatsbyjs.com/plugins/gatsby-plugin-breadcrumb/
